core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -136,17 +136,16 @@ class KiwixTextToSpeech internal constructor(
     // We use JavaScript to get the content of the page conveniently, earlier making some
     // changes in the page
     webView.loadUrl(
-      """
-        javascript:body = document.getElementsByTagName('body')[0].cloneNode(true);
+      "javascript:" +
```

Comment:
is this the fix? it can't load a multiline string? I'd prefer if this were a raw string because it looks much nicer

Reply:
No, In multiline string it doesn't work. I think  It doesn't load readAloud method in a normal string literal.

Reply:
![image](https://user-images.githubusercontent.com/3358282/93075691-106b4880-f67e-11ea-97c2-6916caa0c615.png)

This could be an option to appease my aesthetics

Reply:
It doesn't work.

Reply:
I tried it with multiline text there and it is fine, the problem was mapOf being cast to hashMap I reckon, that is my bad, I gave poor advice on that front

Reply:
@macgills I noticed if I add only one of them (hashMap or multiLinestring) It doesn't work. What should I do?

Reply:
it works for me with only this change to your branch

```
    webView.loadUrl(
      """
        javascript:
        body = document.getElementsByTagName('body')[0].cloneNode(true);
        toRemove = body.querySelectorAll('sup.reference, #toc, .thumbcaption,     title, .navbox');
        Array.prototype.forEach.call(toRemove, function(elem) {    elem.parentElement.removeChild(elem);});
        tts.speakAloud(body.innerText);
        """.trimIndent()
    )
  }
```